uber/utils.py

```diff
@@ -285,3 +285,19 @@ def mount_site_sections(module_root):
     for section in sections:
         module = importlib.import_module(basename(module_root) + '.site_sections.' + section)
         setattr(Root, section, module.Root())
+
+
+def build_uber_absolute_url(relative_uber_page_url):
```

Comment:
OK, I see it in the bands plugin now.

I don't love this function name. Can we change it to something like `convert_to_absolute_url` and have an optional URL_BASE input? So like: `def convert_to_absolute_url(relative_url, base_url = c.URL_BASE)`